unsupported/Eigen/CXX11/src/Tensor/TensorContractionGpu.h

```diff
@@ -542,7 +546,45 @@ EigenFloatContractionKernelInternal16x16(const LhsMapper lhs, const RhsMapper rh
     results[i].x = results[i].y = results[i].z = results[i].w = 0;
   }
 
+#if defined(EIGEN_HIPCC)
 
+#define prefetch_lhs(reg, row, col)                            \
```

Comment:
@deven-amd this block is the prefetch implementation for ROCm, right?

Reply:
correct.

The `prefetch_lhs` macro is defined for both HIP and CUDA, but implementations defer as we do not seem to have support for `loadPacket` in HIP